tools/st2rc.sh

```diff
@@ -0,0 +1,15 @@
+echo -n "Please enter your StackStorm username: "
+read ST2_USERNAME
+export ST2_USERNAME=$ST2_USERNAME
+
+echo -n "Please enter your StackStrom password: "
+read -s ST2_PASSWORD
+echo
+export ST2_PASSWORD=$ST2_PASSWORD
+
+export ST2_BASE_URL=http://127.0.0.1
+export ST2_AUTH_URL=http://127.0.0.1:9100
+export ST2_API_URL=http://127.0.0.1:9101/v1
+
+TOKEN=`st2 auth $ST2_USERNAME -p $ST2_PASSWORD -j | jq '.token'`
```

Comment:
This requires jq to be installed and available.

auth command in master now includes -t/ --only-token flag so you can use that here and then we don't need jq.

(re-posting comment from #1410)